### ISSUE TITLE
Notifications on query errors includes the character name twice and appear erratically

### ISSUE THREAD

**Issue by wbSD** [peterhaneve/evemon#126]
First the simple one.
Many of the notifications have been changed recently (ESI rewrite) to include the character name in the notification text.
I'll use the assets query failing as an example. The text was moved to Resources.resx in this commit, which also added "for {0}." to append the character name,
https://github.com/peterhaneve/evemon/commit/e46203573e9285b68fd447d82b81e5a6c0ca072f#diff-b92199281651586115b0a2d9805c9461R582

However, the name is already added here in the main window code that shows the notification popups:
https://github.com/peterhaneve/evemon/blob/3ccab6321592481ac51c8677808f6d8ee87cddb1/src/EVEMon/MainWindow.cs#L830-L846

The result is that the notification text shows the character name twice, "bla bla of character of character."
![image](https://user-images.githubusercontent.com/1277455/49611151-faa4fb80-f9a0-11e8-9949-e23fb89f4f47.png)

This should be fairly easy to fix by just modifying the Resx file and maybe the notification eventargs to remove the unnecessary string.Format.

Secondly, if one query error occurs, then any subsequent query does not show any notification popups, even if it's a different type.
For example, if you try to query Assets and it fails, then if Contracts fail, you are not notified.
Not until a successful API call has completed for the character is another error notification shown.
I'm not sure if it's intentional like that or not (maybe it was less of an issue pre-ESI because it was just one API request, now it's multiple), but it looks like it's easy to change to notify each different type of query error once, instead of only one.
In the ShouldNotifyError method of CCPCharacter, if m_errorNotifiedMethod is changed to a list of enums (methods) and the input enum is added/removed as required, it makes the error notifications show once per method, until that method has a successful query, which then resets it.

https://github.com/peterhaneve/evemon/blob/a46b476cf948a95f016d5c59ff509c1663487051/src/EVEMon.Common/Models/CCPCharacter.cs#L563-L578

**Comment by peterhaneve** [comment]
I am leaning towards "won't fix" for the second issue, as the intent is to avoid spamming popups if ESI is malfunctioning or the network is experiencing issues.

**Comment by wbSD** [comment]
Sure, I was just brainstorming. The double name thing warrants some changes though

**Comment by peterhaneve** [comment]
So there was one place where the character name was missed, and I decided to nip the issue in the bud and "fix" it everywhere. However, the messages shown in the tooltip apparently were auto-suffixed elsewhere (and since I only checked the popup in the UI, this went unnoticed).
